env.go

```diff
@@ -220,8 +220,27 @@ func EnvDirFromProxyLink(executable string) (string, error) {
 	if filepath.Base(last) != "hermit" {
 		return "", errors.Errorf("binary is not a Hermit symlink: %s", links[0])
 	}
-	envDir := filepath.Dir(filepath.Dir(last))
-	return envDir, nil
+	last = filepath.Dir(last)
+	if filepath.Base(last) != "bin" {
+		return "", errors.Errorf("Hermit not in a bin directory: %s", links[0])
+	}
```

Comment:
@alecthomas I added this belt-and-suspenders check. Thoughts?

Reply:
Seems reasonable to me 👍